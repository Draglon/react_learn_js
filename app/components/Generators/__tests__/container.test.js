import React from 'react';
import configureStore from 'redux-mock-store';

import GeneratorsContainer from '../container';

describe('Generators container', () => {
  const store = configureStore()();
  store.dispatch = jest.fn();

  const defaultProps = {
    store,
  };

  const wrapper = shallow(<GeneratorsContainer {...defaultProps} />);
  const container = wrapper.dive();

  it('renders Generators component', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
