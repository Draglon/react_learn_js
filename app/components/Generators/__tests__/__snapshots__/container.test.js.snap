// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Generators container renders Generators component 1`] = `
<GeneratorsComponent
  calendar={Array []}
  onPlay={[Function]}
  store={
    Object {
      "clearActions": [Function],
      "dispatch": [MockFunction],
      "getActions": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
/>
`;
